weasyl/errorcode.py

```diff
@@ -107,6 +111,7 @@
     "passwordIncorrect": "The password you entered was incorrect.",
     "passwordInsecure": "Passwords must be at least 10 characters in length.",
     "passwordMismatch": "The password you entered did not match the password confirmation.",
+    'permission': permission,
```

Comment:
This already exists as `"InsufficientPermissions"`; can you merge it into `"InsufficientPermissions"`? (I’d like to do it in that direction to avoid cluttering the diff because there are a fair number of existing `raise`s for it.)

Reply:
Yup, going to fix that on my other pull request as well. 
Must have had some tunnel vision going on when doing that :)